.gitattributes

```diff
@@ -1,8 +1,12 @@
 # `git apply` and friends don't understand CRLF, even on windows. Force those
 # files to be checked out with LF endings even if core.autocrlf is true.
-*.patch text eol=lf
+text eol=lf
```

Comment:
The [format](https://git-scm.com/docs/gitattributes) of the lines in this file is `pattern attr1 attr2 …`. If there’s no pattern, the line should just be deleted, rather than moving the first attr into the pattern position.

```suggestion
```